charts/tidb-drainer/templates/scripts/_start_drainer.sh.tpl

```diff
@@ -27,7 +27,7 @@ done
 /drainer \
 -L={{ .Values.logLevel | default "info" }} \
 -pd-urls=http://{{ .Values.clusterName }}-pd:2379 \
--addr=`echo ${HOSTNAME}`.{{ include "drainer.name" . }}:8249 \
+-addr=:8249 \
```

Comment:
able to port forward  to 8249 (k8s [hard coded host as localhost](https://github.com/kubernetes/kubernetes/blob/928817a26a84d9e3076d110ea30ba994912aa477/pkg/kubelet/dockershim/docker_streaming_others.go#L47))